ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot

```diff
@@ -384,7 +385,7 @@ Verify Model Can Be Served And Query On A GPU Node
 Verify Non Admin Can Serve And Query A Model
     [Documentation]    Basic tests leveraging on a non-admin user for preparing, deploying and querying a LLM model
     ...                using Kserve and Caikit+TGIS runtime
-    [Tags]    ODS-2326    WatsonX
+    [Tags]    Smoke    ODS-2326
```

Comment:
Missing TierX tag

Reply:
I thought we don't set Tier tag on Smoke, am I wrong?

Reply:
I believe TierX has to be always included, independently if the test is in Smoke or Sanity or any other special purpose quality gate